samples/cds-sample-application/src/business-partner/business-partner-service-handler.ts

```diff
@@ -0,0 +1,22 @@
+import { BusinessPartner } from '@sap/cloud-sdk-vdm-business-partner-service';
+
+const destinationUrl: string = process.env.CLOUD_DESTINATION_URL;
```

Comment:
do not user the URL. Just use {destinationName: "myDestinationName"} and see below in test.